crates/biome_js_analyze/tests/specs/nursery/noUselessLoneBlockStatements/invalidModule.js

```diff
@@ -0,0 +1,33 @@
+export {}
+
+{}
+
```

Comment:
I forgot about that: we could also add examples with comments to see how it is removed or kept.

Reply:
Good idea, let me add some in both cjs / module files.